src/custom-types.json

```diff
@@ -140,6 +140,33 @@
   "application/vnd.android.package-archive": {
     "compressible": false
   },
+  "application/vnd.apple.pages": {
+    "compressible": false,
+    "extensions": ["pages"],
+    "sources":[
+      "https://codex.wordpress.org/Function_Reference/get_allowed_mime_types",
+      "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+      "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
+    ]
+  },
+   "application/vnd.apple.numbers": {
+    "compressible": false,
+     "extensions": ["numbers"],
+     "sources":[
+       "https://codex.wordpress.org/Function_Reference/get_allowed_mime_types",
+       "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+       "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
+      ]
+  },
+   "application/vnd.apple.keynote": {
+    "compressible": false,
+     "extensions": ["key"],
+     "sources":[
+       "https://codex.wordpress.org/Function_Reference/get_allowed_mime_types",
+       "https://docs.alfresco.com/5.2/references/valid-transformations.html",
+       "https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html",
```

Comment:
None of these are a primary reference. Who owns this file extension? I believe Apple, right? Please like to something Apple controls.

Reply:
apologies for the delayed response. In the primary link i provided it has all three extensions. Its from Apache.
https://tika.apache.org/1.15/api/org/apache/tika/parser/iwork/IWorkPackageParser.html

Reply:
Unless you're saying that Apache created IWorks, then it's not a primary source.

Reply:
After a day of talking to Apple Support teams i was able to get a reference. Le me know if this can be accepted. I have submitted a request to add in IANA list as well. 

https://developer.apple.com/library/archive/qa/qa1630/_index.html

Reply:
Nice! Where are the MIME types on the page? I don't see them for some reason.

Reply:
Its under the line "iPhone OS 3.0 supports these additional document types:"  This is the closes thing Apple team could find in their documents.

Reply:
Rich Text Format (.rtf)
Rich Text Format Directory (.rtfd.zip)
Keynote '09 (.key)
Numbers '09 (.numbers)
Pages '09 (.pages)

Is what I see there. Those are just file extension, not the MIME types.

Reply:
Yes this is the closest thing they have. Thanks for your time. I will update if i find more information from Apple otherwise i will get it reviewed by IANA team.